include/tvm/ir/name_supply.h

```diff
@@ -54,9 +54,11 @@ class NameSupplyNode : public Object {
    * \brief Generates a unique name from this NameSupply.
    * \param name The name from which the generated name is derived.
    * \param add_prefix If set to true, then the prefix of this NameSupply will be prepended to the
-   * name. \return A unique name.
+   * name.
+   * \param add_underscore If set to true, add '_' between prefix and a digit.
+   * \return A unique name.
    */
-  String FreshName(const String& name, bool add_prefix = true);
+  String FreshName(const String& name, bool add_prefix = true, bool add_underscore = true);
```

Comment:
This change is only for backward-compat according to the variable naming convention of `BlockBuilder` (`lv1` vs `lv_1`). `NameSupply` adds an underscore before a digit by default. 

If we are ok with introducing such trivial variable naming change, I'm more than happy to remove this diff.

Reply:
I think no suffix is better overall for UX, we can upstream this change